worlds/noita/__init__.py

```diff
@@ -0,0 +1,111 @@
+import string
+from typing import Dict, List
+from .Items import NoitaItem, item_table, item_pool_weights
+from .Locations import NoitaLocation, item_pickups
+
+from BaseClasses import Region, RegionType, Entrance, Item, ItemClassification, MultiWorld, Tutorial
+from .Options import noita_options
+from worlds.AutoWorld import World, WebWorld
+
+client_version = 1
+
+
+class NoitaWeb(WebWorld):
+    tutorials = [Tutorial(
+        "Multiworld Setup Guide",
+        "A guide to setting up the Noita integration for Archipelago multiworld games.",
+        "English",
+        "setup_en.md",
+        "setup/en",
+        ["DaftBrit"]
+    )]
+    theme = "partyTime"
+
+
+class NoitaWorld(World):
+    """
+    Noita is a magical action roguelite set in a world where every pixel is physically simulated. Fight, explore, melt,
+    burn, freeze and evaporate your way through the procedurally generated world using spells you've created yourself.
+    """
+    game: str = "Noita"
+    option_definitions = noita_options
+    topology_present = False
+
+    item_name_to_id = item_table
+    location_name_to_id = item_pickups
+
+    data_version = 5
+    forced_auto_forfeit = True
+    web = NoitaWeb()
+
+    def generate_basic(self):
+
+        if self.world.bad_effects[self.player].value:
```

Comment:
```suggestion
        if self.world.bad_effects[self.player]:
```
Option classes have methods for handling comparisons like this so better to not use value here.

Reply:
You could actually just do `pool_option = self.world.bad_effects[self.player].value` if you like as `value` in this case is an integer of either 0 or 1. You can then either invert how you use pool_option or how your options are laid out.

Reply:
Thanks @alwaysintreble appreciate the feedback, I'll take a look this evening